src/sentry/static/sentry/app/views/releases/detail/utils.tsx

```diff
@@ -86,3 +96,32 @@ export function getReposToRender(repos: Array<string>, activeRepository?: Reposi
   }
   return [activeRepository.name];
 }
+
+/**
+ * Get high level transaction information for this release
+ */
+export function getReleaseEventView(
+  selection: GlobalSelection,
+  version: string,
+  organization: LightWeightOrganization
+): EventView {
+  const {projects, environments, datetime} = selection;
+  const {start, end, period} = datetime;
+
+  const discoverQuery = {
+    id: undefined,
+    version: 2,
+    name: `${t('Release Apdex')}`,
+    fields: [`apdex(${organization.apdexThreshold})`],
+    query: stringifyQueryObject(
+      new QueryResults([`release:${version}`, 'event.type:transaction', 'count():>0'])
```

Comment:
Is the `count():>0` necessary here?

Reply:
Yeah, this is a protection against getting apdex==NaN from snuba when the release doesn't have any transactions yet.